source/scpd/types/Stellar_SCP.d

```diff
@@ -321,41 +321,42 @@ struct SCPQuorumSet {
     import std.conv;
 
     const qc1 = toSCPQuorumSet(QuorumConfig(2,
-        [PublicKey.fromString("GBFDLGQQDDE2CAYVELVPXUXR572ZT5EOTMGJQBPTIHSLPEOEZYQQCEWN"),
-         PublicKey.fromString("GBYK4I37MZKLL4A2QS7VJCTDIIJK7UXWQWKXKTQ5WZGT2FPCGIVIQCY5")]));
+        [PublicKey.fromString("boa1xrv266magda8dc87uche9zvj8842shz3sdavw0qecpgeykyv4ynss5caytq"),
+         PublicKey.fromString("boa1xrv266magd38dc87uche9zvj8842shz3sd3vw0qecpgeykyv4ynssw6rsgy")]));
 
     assert(qc1.hashFull() == Hash.fromString(
-        "0x7b56982f02dfbf3737ff560fe6511674e182d38475f0516cb8b0f338a3156543b0731e66c9a3ced01c238652b12e51c95207ec2bf6eae237f24b08a357a1bd2a"),
+        "0x2a37ad395b2c50d18f2a52e4b6759a8bc2e43f7835ef5f41805d1f717e528943f0de5a15ec9bf4916db6da76f4d1f04e7df0bfdd5d5973a734acce115f07ed67"),
```

Comment:
Why did the hash change ? The quorum set hash only depends on the binary representation of the public key, doesn't it ?

Reply:
Will update.